Source/WebCore/html/canvas/GPUCanvasContextCocoa.h

```diff
@@ -111,9 +121,13 @@ class GPUCanvasContextCocoa final : public GPUCanvasContext {
 
     GPUIntegerCoordinate m_width { 0 };
     GPUIntegerCoordinate m_height { 0 };
-    float m_contentsHeadroom { 0.f };
+#if HAVE(SUPPORT_HDR_DISPLAY)
+    bool m_suppressEDR { false };
```

Comment:
Very minor but can we move this with the other `bool` please (`m_compositingResultsNeedsUpdating`) to avoid unnecessary padding between member variables?